src/main/java/net/minecraftforge/registries/tags/ITag.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Minecraft Forge - Forge Development LLC
+ * SPDX-License-Identifier: LGPL-2.1-only
+ */
+
+package net.minecraftforge.registries.tags;
+
+import net.minecraft.tags.TagKey;
+
+import java.util.stream.Stream;
+
+/**
+ * A tag is a collection of elements with an identifying {@link TagKey tag key}.
+ * For Forge, these are bound on world load.
+ * Tags will always be empty until they are bound.
+ * A tag instance provided for a given {@link TagKey} from a given {@link ITagManager} will always return the same instance on future invocations.
```

Comment:
Is this statement of same instance also true for if a reload occurs? Either way it is probably worth clarifying it.

Reply:
Same forge registry instance means same tag manager and same tags. Reload doesn't recreate these registries afaik.